os.md

```diff
@@ -31,3 +31,9 @@ Ubuntu 22.04 | All supported kernels
 Ubuntu 20.04 | >= 5.4.0-1047-aws, >= 5.8.0-1034-aws, >= 5.11.0-1009-aws
 Ubuntu 18.04 | >= 4.15.0-1101-aws, >= 5.4.0-1047-aws
 Redhat Entreprise Linux 8 | >= 4.18.0-305.10
+
+# Operating Systems not supported by Graviton based instances
+
+ Name | Version | Support | Comment
+------ | ------ | ----- | ----- | 
+Microsoft Windows | All versions | No |  Currently, Graviton instances do not support Microsoft Windows; however, this might change in the future.
```

Comment:
It's redundant to include a "support" column under a section exclusively for OSes without support. Remove "however, this might change in the future." Windows support for Graviton is entirely up to Microsoft. Customers who want Windows support for Graviton should ask Microsoft to enable it.

Reply:
done